docs/src/main/sphinx/release/release-375.md

```diff
@@ -0,0 +1,81 @@
+# Release 375 (25 Mar 2022)
+
+## General
+
+* Change behavior of `ALTER TABLE qualified_name TO unqualified_name` to leave
+  the table in the schema. This is backwards-incompatible behavioral change. ({issue}`11282`)
+* Disallow table redirections for the `GRANT`, `REVOKE`, and
+  `SET TABLE AUTHORIZATION` table tasks. ({issue}`11302`)
+* Improve performance of queries that contain filter predicates on aggregation
+  results. ({issue}`11469`)
+* Fix `SHOW CREATE TABLE` to show actual table name in case of table
+  redirections, so that the statement can be used to recreate the table. ({issue}`11604`)
+* Fix scheduling for non-remotely accessible splits in fault-toleration execution. ({issue}`11581`)
+* Fix incorrect `is_nullable` value in `information_schema.columns` table. ({issue}`11613`)
+
+## JDBC driver
+
+* Allow `PreparedStatement.close()` to be called multiple times. ({issue}`11620`)
+* Fix incorrect `is_nullable` value in `DatabaseMetaData.getColumns()` method. ({issue}`11613`)
+
+## Cassandra connector
+
+* Return `0` instead of row count in completed bytes. ({issue}`11644`)
+
+## Delta Lake connector
+
+* Add access control to `drop_extended_stats` and `vacuum` procedures. ({issue}`11633`)
+* Fix incorrect query results when previous query coincides with drop table. ({issue}`11562`)
+
+## Hive connector
+
+* Fix infinite loop in the query optimizer when query contains predicates on a
+  struct field. ({issue}`11559`)
+* Fix query failure when reading a partitioned table with a predicate on a
+  partition column with a specific name such as `table`, `key`, `order`, and
+  others. ({issue}`11512`)
+
+## Iceberg connector
+
+* Fix query failure when query contains predicates on a struct field. ({issue}`11560`)
+* Fix query failure when reading from `$files` system table after a table column
+  has been dropped. ({issue}`11576`)
+
+## Kudu connector
+
+* Improve write performance by flushing operations in batches. ({issue}`11264`)
+* Fix race in creating schemas table if schema emulation is enabled. ({issue}`11264`)
+
+## MongoDB connector
+
+* Support predicate pushdown on `boolean` columns. ({issue}`11536`)
+
+## MySQL connector
+
+* Add support for table comments. ({issue}`11211`)
+
+## Pinot connector
+
+* Fix handling of passthrough queries that include expressions containing
```

Comment:
Simplify to “queries that contain aggregation functions”

Reply:
done